appcelerator-archive/amp
# Disable notifications

Resolves #1177 

## How to test
```
$ ampmake build
$ alias amp='amp -s localhost'
$ amp cluster create --tag=local --registration=none --notifications=false
$ amp user signup --name user --password password --email email@user.amp
$ amp login --name user --password password
$ amp org create --org org --email email@org.amp
$ amp logs -i amp_amplifier | grep Notifications
[localhost:50101]
Notifications: false
```
You should not receive an emails during that test.
